main.tf

```diff
@@ -282,7 +299,6 @@ resource "kubernetes_secret" "secret" {
   }
 
   data = {
-    "MYSQL"       = var.database_connection_string
-    "OIDC_SECRET" = var.oidc_secret
```

Comment:
before this PR, looks like these secrets were not explicitly set in the environment, if that's the case, how did mysql work? Was something just broken?

Reply:
ah, we gotta remove the static secrets OIDC_SECRET and MYSQL from lines 97 and 127. 
https://github.com/wandb/terraform-kubernetes-wandb/pull/35/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR97
and

Reply:
Hmm, did that work?  I think you merged that PR into this one but it's still showing the wrong change against master.